pipenv/core.py

```diff
@@ -791,7 +791,7 @@ def do_install_dependencies(
     skip_lock=False,
     concurrent=True,
     requirements_dir=None,
-    pypi_mirror=False,
+    pypi_mirror=None,
```

Comment:
This probably shouldn't need to change

Reply:
Why the `pypi_mirror` becoms False here but None in the rest parts of codes?

Reply:
Most likely there is no good reason for this -- and apologies for not taking the time to look at it more closely! If, as you say, this is `None` everywhere else, and we don't differentiate in the function below (e.g. by checking if it's `False`) then it will be ok to make this change for consistency